### PR TITLE
On new Conan version, update repository files [skip ci]

### PR DESCRIPTION
**Pull request by jgsogo**
On new Conan version, update repository files (recovered from https://github.com/conan-io/conan-docker-tools/commit/0533893a4de71a5e5442418fd74ba26396fdbe5d)

### CODE REVIEW

File: .ci/on_conan_release.jenkinsfile [hunk 1]
```diff
@@ -26,4 +26,72 @@ node('Linux') {
...
+        // Open PR in Github
+        stage('Open PR in Github') {
+            def data = readJSON text: '{}'
+            data.title = "[up conan] Update Conan version (${params.conan_version})" as String
+            data.head = branchName
+            data.base = 'master' as String
+            data.body = '''\
+                New Conan version released.
+
+                Please, @conan-io/barbarians have a look to this PR
```
Comment (uilianries) [inline .ci/on_conan_release.jenkinsfile:81]: I think we could create a docker group. Barbarians will alert all conan team.

**Review by uilianries — APPROVED** [review]
LGTM

File: .ci/on_conan_release.jenkinsfile (hunk 1 above)
Comment (jgsogo) [reply .ci/on_conan_release.jenkinsfile:81]: We can ask for actual reviews from certain users (I'll do that)

Comment (uilianries) [reply .ci/on_conan_release.jenkinsfile:81]: nice!

**Pull request merged**